helparty/src/main/java/com/hamryt/helparty/service/LoginServiceImpl.java

```diff
@@ -14,25 +13,22 @@
 public class LoginServiceImpl implements LoginService {
 
     private final Encryptor encryptor;
-    private final UserMapper userMapper;
     private final UserService userService;
+    private final HttpSession session;
 
     @Transactional
-    public UserDto login(String email, String password, HttpSession session) {
+    public UserDto login(String email, String password) {
 
         String encryptPassword = encryptor.encrypt(password);
+        UserDto userDto = userService.findUserByEmailAndPassword(email, encryptPassword);
 
-        if (userService.findByEmailAndPassword(email, encryptPassword)) {
-            setLoginUserEmail(session, email);
-        } else {
+        if(userDto!=null){
+            session.setAttribute(SessionKeys.LOGIN_USER_EMAIL, userDto.getEmail());
+        }else{
```

Comment:
이런 띄어쓰기 컨벤션도 계속 주의해주세요~ checkstyle도 적용해봐주시고요~

```suggestion
        if (userDto!=null){
            session.setAttribute(SessionKeys.LOGIN_USER_EMAIL, userDto.getEmail());
        } else {
```

Reply:
넵 주의하겠습니다.